requirements.txt

```diff
@@ -1,4 +1,5 @@
-django>=1.5.1,<=1.6.7
+django>=1.5.1,<=1.6.8
 wheel==0.23.0
 oauth2==1.5.211
-git+git://github.com/odero/django-uuidfield.git#egg=django-uuidfield
+requests==2.1
+django-uuidfield==0.5.0
```

Comment:
There is a bug in this version of django-uuidfield which is why I hard to switch to the forked git version